contrib/sbase/Makefile

```diff
@@ -0,0 +1,312 @@
+TOPDIR = $(realpath ../..)
+
+.SUFFIXES:
```

Comment:
Why do you need `.SUFFIXES` rule?

Reply:
I think it's just a leftover after some initial work... I'll get rid of it